R/stan_module.R

```diff
@@ -49,17 +49,15 @@ read_stan <- function(string) {
 setMethod(
     f = "initialize",
     signature = "StanModule",
-    definition = function(
-        .Object,
-        ...,
-        functions = "",
-        data = "",
-        parameters = "",
-        transformed_parameters = "",
-        generated_quantities = "",
-        priors = list(),
-        inits = list()
-    ) {
```

Comment:
why has this been changed ?

Reply:
`Styler` changed the syntax. The arguments and the function are the same.